src/server/models/app-models.js

```diff
@@ -339,11 +344,11 @@ export default function (db) {
   }
 
   AuditEvent.belongsTo(User, {
-    onDelete: 'SET NULL',
+    constraints: false,
```

Comment:
:ok_hand:

Reply:
I only now realized that #265 also created foreign keys for `presence_histories.userId` with `onDelete: 'SET NULL',`. Should I make that also not use foreign keys like this one?

Reply:
In that case we probably do want to keep the foreign key, and I think set null is a good behavior for that case